library/bignum_core.h

```diff
@@ -6,6 +6,11 @@
  *  modules should use the high-level modular bignum interface (bignum_mod.h)
  *  or the legacy bignum interface (bignum.h).
  *
+ * This module is about processing non-negative integers with a fixed upper
+ * bound that's of the form 2^#biL-1. Many operations treat these numbers
```

Comment:
Did you mean `2^{limbs*#biL-1}`?

Reply:
2^{limbs*biL}-1, actually. I clarified, and defined the word limbs which was previously implicit.